libcaf_net/caf/net/multiplexer.hpp

```diff
@@ -60,179 +31,81 @@ class CAF_NET_EXPORT multiplexer : public detail::atomic_ref_counted,
 
   // -- constructors, destructors, and assignment operators --------------------
 
-  ~multiplexer();
+  virtual ~multiplexer();
```

Comment:
We already have a base type with a (virtual) destructor.

```suggestion
  ~multiplexer() override;
```

Reply:
Excellent point. Thanks.